dbt/adapters/dremio/credentials.py

```diff
@@ -58,22 +64,25 @@ def unique_field(self):
 
     def _connection_keys(self):
         # return an iterator of keys to pretty-print in 'dbt debug'
-        # raise NotImplementedError
-
         return (
-            "driver",
             "cloud_host",
             "cloud_project_id",
             "software_host",
             "port",
+            "username",
             "UID",
-            "database",
-            "schema",
-            "additional_parameters",
-            "datalake",
-            "root_path",
+            "additional_parameters",  # TODO: investigate
             "environment",
             "use_ssl",
+            "object_storage_source",
+            "space_root_folder",
+            "root_path",
+            "datalake",
```

Comment:
Related to previous comment about putting aliases in the same place as originals. We could remove datalake, database, root_path, and schema.

Reply:
This one I am going to leave in, because it can be sued to verify that our aliases are working. I will add a comment to make that clear.